File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int answer = 0;
        int length = Integer.toString(n).length();
        for (int i = 0; i < length; i++) {
            answer = answer + (n % 10);
            n /= 10;
        }

        return answer;
    }
}

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.kt
class Solution {
    fun solution(n: Int): Int {
        var answer = 0
        val list: List<String> = n.toString().split("");
        for (numStr in list) {
            if (numStr.isEmpty()) continue
            else answer += Integer.parseInt(numStr)
        }
        
        return answer
    }
}